clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -78,3 +78,14 @@ endmacro()
 # Always set SYCL_HAS_FP16 to true to assume SYCL device to support float16
 message("dpct.cmake: SYCL_HAS_FP16 is set true by default.")
 set(SYCL_HAS_FP16 TRUE)
+
+# Link MKL library to target
+macro(DPCT_HELPER_ADD_MKL_TO_TARGET target)
```

Comment:
Pls add behavior test in SYCLomatic-test repo by following https://github.com/oneapi-src/SYCLomatic-test/tree/SYCLomatic/behavior_tests/src/cmake_dpct_helper_compile_sycl_code

Also add sycl test in https://github.com/intel-sandbox/cmake_script_migration/tree/master/lit/target_link_libraries/

Reply:
Raised the behaviour test https://github.com/oneapi-src/SYCLomatic-test/pull/573 for dpct_add_mkl_to_target